frontend_lib/src/component/Tags/TagList.styl

```diff
@@ -0,0 +1,96 @@
+@import '../../css/Variable.styl'
+
+.taglist
+  flex-direction column
+  width calc(100% - 65px)
+  height 100%
+  min-height 260px
+  overflow auto
+  margin-bottom 10px
+  &__header
+    margin 10px 15px
+    padding 10px
+    font-size 20px
```

Comment:
Can you use the font size defined at Variable.styl (valid to all font-size in this file)?

Reply:
fixed